pkgs/by-name/ue/uemacs/package.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, stdenv
+, fetchgit
+, ncurses
+, pkg-config
+}:
+
+stdenv.mkDerivation {
+  pname = "uemacs";
+  version = "2018-07-19";
+
+  src = fetchgit {
+    url = "https://git.kernel.org/pub/scm/editors/uemacs/uemacs.git";
+    rev = "1cdcf9df88144049750116e36fe20c8c39fa2517";
+    hash = "sha256-QSouqZiBmKBU6FqRRfWtTGRIl5sqJ8tVPYwdytt/43w=";
+  };
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [
+    ncurses
+  ];
+
+  postPatch = ''
+    substituteInPlace Makefile --replace "lcurses" "lncurses"
+    substituteInPlace Makefile --replace "/usr/bin" "$out/bin"
+    substituteInPlace Makefile --replace "/usr/lib" "$out/share"
+
+    substituteInPlace epath.h --replace "/usr/global/lib/" "$out/share/"
+  '';
+
+  installPhase = ''
+    mkdir -p $out/{bin,share}
+    make install
+  '';
+
+  meta = with lib; {
+    description = "Linus Torvalds's random version of microemacs with my private modificatons";
+    homepage = "https://git.kernel.org/pub/scm/editors/uemacs/uemacs.git/about/";
+    license = licenses.free;
```

Comment:
where did you get the license info from, couldn't find anything in the repo except for the part in the readme which wouldn't allow for commercial use before asking for permissions, so don't know about the licenses.free part.

Reply:
I haven't found a better fitting license in our licenses :woman_shrugging:

Reply:
Changed to `unfree` since all non-commercial licenses in `lib.licences` (eg creative commons ones) set `free` to `false`.